lib/cli/triage.js

```diff
@@ -47,21 +47,50 @@ module.exports = async () => {
     return resolvePlatformCli();
   }
 
-  if (
-    (
-      await Promise.all(
-        ['yml', 'yaml', 'json', 'js', 'ts'].map(async (extension) => {
-          try {
-            await fsp.access(`serverless-compose.${extension}`);
-            return true;
-          } catch {
-            return false;
-          }
-        })
-      )
-    ).some(Boolean)
-  ) {
-    return '@serverless/compose';
+  const isComposeIgnoredCommand = (() => {
+    const args = process.argv.slice(2);
+    // Simplified check - if arg starts with `-` then we consider it to be a param and everything before it to
+    // be a command
+    const firstParamIndex = args.findIndex((element) => /^-/.test(element));
```

Comment:
I think there's no need for regex, `.startsWith` should do the job

Reply:
good call - I've started with more complex regex and simplified it to a point where `startsWith` does the trick as you suggested 👍 changed